civis/io/_files.py

```diff
@@ -76,7 +97,11 @@ def file_to_civis(buf, name, api_key=None, client=None, **kwargs):
         # This streams from the open file buffer without holding the
         # contents in memory.
         en = MultipartEncoder(fields=form_key)
-        if en.len / 2 ** 20 < 100:
+        # The refusal error from AWS states 5368730624 is the max size allowed
+        if en.len >= 5 * 2 ** 30:  # 5 GB
+            msg = "Cannot uploaded files greater than 5GB. Got {:d}."
```

Comment:
"uploaded" --> "upload"

Does the AWS error happen immediately, or only after the user has uploaded 5 GB? If it's immediate, it seems better to let the attempt go through and show the user AWS's error message.